bmrg-flow/README.md

```diff
@@ -124,13 +126,12 @@ The following table lists the configurable parameters of the chart and their def
 | `mongodb.password` | Database user password | `passw0rd` |
 | `database.mongodb.secretName` | The secret to get the password from. | |
 
-### NATS Configuration
+### Eventing Configuration
 
 | Parameter | Description | Default Value |
 |---|---|---|
-| `nats.enable` | Enable NATS | `true` |
-| `nats.url` | The NATS Url | `nats://bmrg-nats:4222` |
-| `nats.cluster` | NATS Streaming Cluster | `bmrg-stan` |
+| `eventing.enables` | Enables eventing as part of the installation. This will install the listener service to accept incoming webhooks and events. Optionally requires NATS | `true` |
```

Comment:
I think this would be `eventing.enabled`

Reply:
Typo. Thanks!